assembl/static2/js/app/components/debate/brightMirror/instructionsText.jsx

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React from 'react';
+import { Col, Grid } from 'react-bootstrap';
+import activeHtml from 'react-active-html';
+import { postBodyReplacementComponents } from '../common/post/postBody';
+
+export type InstructionsTextProps = {
+  /** Instruction title */
+  title: string,
+  /** Instruction body */
+  body: string
+};
+
+const renderRichtext = text => activeHtml(text, postBodyReplacementComponents());
```

Comment:
Is this function specific to bright mirror rich text ? Otherwise I guess we could make use of it elsewhere. If so export it from globalFunctions